pkg/limayaml/defaults.go

```diff
@@ -334,20 +334,28 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 	iface := make(map[string]int)
 	for _, nw := range append(append(d.Networks, y.Networks...), o.Networks...) {
 		if i, ok := iface[nw.Interface]; ok {
-			if nw.VNL != "" {
-				networks[i].VNL = nw.VNL
-				networks[i].SwitchPort = nw.SwitchPort
+			if nw.VNLDeprecated != "" {
+				networks[i].VNLDeprecated = nw.VNLDeprecated
+				networks[i].SwitchPortDeprecated = nw.SwitchPortDeprecated
+				networks[i].Socket = ""
+				networks[i].Lima = ""
+			}
+			if nw.Socket != "" {
+				networks[i].Socket = nw.Socket
+				networks[i].VNLDeprecated = ""
+				networks[i].SwitchPortDeprecated = nw.SwitchPortDeprecated
```

Comment:
Doesn't really matter because `SwitchPortDeprecated` shouldn't get used at all when `VNLDeprecated` is not set, but this seems more apppropriate:

```suggestion
				networks[i].SwitchPortDeprecated = 0
```

Reply:
Fixed